src/chrome/komodo/locale/en-US/komodo.dtd

```diff
@@ -931,4 +931,4 @@
 <!ENTITY contributePackages.label "Contribute Packages">
 <!ENTITY shareTool.label "Share this Tool">
 
-
+<!ENTITY tabAwarePaste.label "Tab-Aware Paste">
```

Comment:
Can you please rename it to "Paste (Preserve Indentation)"

Spaces all the way, but also we shouldn't judge .. :p

Reply:
That's a long label for a context menu, to be fair.

Reply:
It really isn't. Have a look at your context menu's.